install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -5,7 +5,6 @@
   },
   "staticResources": {
     "listeners": [
-      {{- if and .Values.prometheus.enabled .Values.envoy.prometheus.enabled }}
       {
         "name": "envoy-prometheus-metrics-listener",
```

Comment:
we can rename this listener to something more generic, as now /healthz endpoint is part of this listener.